Packs/Aws-SecretsManager/Integrations/AwsSecretsManager/AwsSecretsManager.py

```diff
@@ -0,0 +1,298 @@
+        raise Exception('secret_id is mandatory inorder to run this command!')
+
+    response = client.restore_secret(SecretId=args.get('secret_id'))
+
+    if response['ResponseMetadata']['HTTPStatusCode'] == 200:
+        demisto.results("the secret was restored successfully")
+
+
+def aws_secrets_manager_secret_policy_get_command(client: AWSClient, args: Dict[str, Any]):
+    client = client.aws_session(
+        service=SERVICE,
+        role_arn=args.get('roleArn'),
+        role_session_name=args.get('roleSessionName'),
+        role_session_duration=args.get('roleSessionDuration'),
+    )
+
+    if not args.get('secret_id'):
+        return_error('secret_id is mandatory inorder to run this command!')
+
+    response = client.get_resource_policy(
+        SecretId=args.get('secret_id')
+    )
+
+    readable_output = {'Name': response.get('Name', ''),
+                       'ARN': response.get('ARN', ''),
+                       'Policy': response.get('ResourcePolicy', '')}
+
+    human_readable = tableToMarkdown('AWS Secret Policy', readable_output)
+
+    return_results(CommandResults(
+        outputs_prefix='AWS.SecretsManager.Policy',
+        outputs=response,
+        outputs_key_field='Name',
+        readable_output=human_readable
+    ))
+
+
+def fetch_credentials(client: AWSClient, args: Dict[str, Any]):  # pragma: no cover
+    client = client.aws_session(
+        service=SERVICE,
+        role_arn=args.get('roleArn'),
+        role_session_name=args.get('roleSessionName'),
+        role_session_duration=args.get('roleSessionDuration'),
+    )
+
+    creds_dict = dict()
+    if args.get('secret_id'):
+        try:
+            creds_dict[args.get('secret_id')] = client.get_secret_value(SecretId=args.get('secret_id'))
+        except Exception as e:
+            demisto.debug(f"Could not fetch credentials: {args.get('secret_id')}. Error: {e}")
+    else:
+        for secret in client.list_secrets()['SecretList']:
+            creds_dict[secret.get('Name')] = client.get_secret_value(SecretId=secret.get('Name'))
+
+    credentials = []
+    for cred_key in creds_dict.keys():
+        try:
+            secret_as_dict = json.loads(creds_dict[cred_key].get("SecretString"))
+
+            if not any(key in ["user", "password", "workgroup", "certificate"] for key in secret_as_dict.keys()):
+                credentials.append({
+                    "user": secret_as_dict.get("username", ""),
+                    "password": secret_as_dict.get("password", ""),
+                    "workgroup": secret_as_dict.get("workgroup", ""),
+                    "certificate": secret_as_dict.get("certificate", ""),
+                    "name": f'{creds_dict[cred_key].get("Name")}',
+                })
+            else:
+                LOG(f'({creds_dict[cred_key]}) has no keys supporting the format')
+        except Exception as e:
+            return_error(f'theres is a problem parsing ({creds_dict[cred_key]}) secret value, {e}')
```

Comment:
Please replace this with a raise - `return_error` is expected to run in `main` only.